hazelcast/src/main/java/com/hazelcast/internal/nio/ClassLoaderUtil.java

```diff
@@ -52,6 +54,8 @@
     private static final ClassLoaderWeakCache<Class> CLASS_CACHE = new ClassLoaderWeakCache<Class>();
     private static final Constructor<?> IRRESOLVABLE_CONSTRUCTOR;
 
+    private static final ClassLoader FAKE_CLASSLOADER = new URLClassLoader(new URL[0]);
```

Comment:
Better name: `NULL_FALLBACK_CLASSLOADER`
The classloader isn't _fake_, it's actually being used to load classes.

Also, without explicit parent, the classloader will have `System.getSystemClassLoader()` as parent. It seems to me that we should use `ClassLoaderUtil.class.getClassLoader()` as the parent - this is what `Class.forName()`, that was used before for null CL, would use. Though I'm not sure about the actual difference, seems a safer choice to me.

Reply:
Done